lib/request.js

```diff
@@ -417,7 +417,13 @@ module.exports = {
 
   get subdomains() {
     var offset = this.app.subdomainOffset;
-    return (this.host || '')
+    var ip = /^(\d{1,3}\.){3}\d{1,3}$/gim;
```

Comment:
Isn't this part of the code Node.js `net` library? https://nodejs.org/dist/latest-v4.x/docs/api/net.html#net_net_isip_input